### ISSUE TITLE
Gutenberg JS TypeError: Cannot read property 'title' of undefined

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/ednc-2020#125]
```
TypeError: Cannot read property 'title' of undefined
    at https://ednc.test/wp-includes/js/dist/block-editor.min.js?ver=aacb5c07c68e448af6536df79db8f400:50:137576
    at je (https://ednc.test/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:78:476)
    at qe (https://ednc.test/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:89:115)
    at ph (https://ednc.test/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:216:489)
    at lh (https://ednc.test/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:126:409)
    at O (https://ednc.test/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:121:71)
    at ze (https://ednc.test/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:118:14)
    at https://ednc.test/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:53:49
    at unstable_runWithPriority (https://ednc.test/wp-includes/js/dist/vendor/react.min.js?ver=16.9.0:26:340)
    at Ma (https://ednc.test/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:52:280)
```

**Comment by andrewmarino** [comment]
This is only happening on some posts.

**Comment by andrewmarino** [comment]
Related: https://github.com/unitymakesus/ednc-2020/commit/7267e9d62257fca239bbf6c0984d8e19db96dd3c#diff-af758af3d2f824e8b6b7b10416f063ec

**Comment by isabisa** [comment]
Fixed in 8524245
